participations/locales/en.yml

```diff
@@ -0,0 +1,5 @@
+---
+name: Participations
+description: >
+  All participations made on the platform : proposals, endorsements, proposals' votes, debates, answers to surveys and comments. 
```

Comment:
Add nuance about unregistered user survey answers. 
Add them to the mix using the session id creates potential duplicates that we can't verify.

Reply:
Fixed in 3ac05d8